uber/site_sections/registration.py

```diff
@@ -698,9 +698,6 @@ def register(self, session, message='', error_message='', **params):
                     attendee.paid = c.NEED_NOT_PAY
                 elif attendee.payment_method == c.STRIPE:
                     raise HTTPRedirect('pay?id={}', attendee.id)
-                elif attendee.payment_method == c.GROUP:
-                    message = 'Please proceed to the preregistration line to pick up your badge.'
-                    attendee.paid = c.PAID_BY_GROUP
```

Comment:
TBH we should probably leave this part in.  If the option is removed from the settings then it won't matter, but this makes it easier to add that functionality back in.

Reply:
Hm, I thought leaving this in would cause the system to throw an error, but I just tested it and I guess that isn't always the case. I'd still kind of prefer to not leave fossils all over the code -- we had this option for years and apparently it consistently confused the heck out of attendees, so it doesn't seem particularly likely we'll want it again later.

Reply:
Hmm, fair enough I guess.  I felt like this worked really well when we were a much smaller event though.  But I'd need to ask e.g. the Labs folks how it worked out for them.  No super-strong opinion I guess - mostly it's just weird to be like "but but but when *I* was running reg 10 years ago, this worked great!" :P